### DISCUSSION TITLE
How to programatically zoom and pan <GeoSearch /> ?

### DISCUSSION THREAD

**Discussion by everdrone** [algolia/instantsearch#5307] [Q&A] [answered]
When using `<GeoSearch />`, is it possible to pan and zoom into a `lat, lng` point or a `boundingBox`?

**Comment by francoischalifour** [comment, marked as answer]
If you need to only set the initial position, you can use the [`initialPosition`](https://www.algolia.com/doc/api-reference/widgets/geo-search/react/#widget-param-initialposition) prop.

If you need to programmatically change the position, you would need to use `connectGeoSearch()` with the [`refine()`](https://www.algolia.com/doc/api-reference/widgets/geo-search/react/#connector-param-provided-refine) prop.
